### DISCUSSION TITLE
v2.10.32

### DISCUSSION THREAD

**Discussion by fiftin** [semaphoreui/semaphore#2459] [Announcements]
## Security
* Clear run app's environment (reported by @geckoadi)
* Update packages

## Bugfixes
* Import/export survey variables of Templates
* Handle `<playbook_dir>/requirements.txt` by ansible-galaxy.

## Development
* E2E tests launch for VSCode and DevContainer

<hr /><em>This discussion was created from the release <a href='https://github.com/semaphoreui/semaphore/releases/tag/v2.10.32'>v2.10.32</a>.</em>

**Comment by mfranikowski** [comment]
When I try to use task with a role from my own repository (Gitlab) by defining it in requirements.yml:

```
- name: mycompany.my_role
  src: git+ssh://git@gitlab.mycompany.com/ansible/roles/my_role.git
```

when running the template I get an error message about installing the role with the notice:

```
could not find/use git, it is required to continue with installing
```

In version 2.10.30, this works correctly.

Thanks.

Marcin

**Reply by fiftin** [reply]
Hi @mfranikowski 

Fixed in v2.10.33+
